### PR TITLE
UG changes

### LINKED ISSUE #222: How to navigate section for UG

### LINKED ISSUE #223: Purpose of UG + complete product information

### LINKED ISSUE #228: Candidate section
- add_c: convert the format to regular text
- status bullet point should be different from the different statuses
- make it clear that the status is different in position and candidate
- make it clear which fields are mandatory
- example to help explain is helpful
- put list before delete
- remark: maybe add a note that there is no limit
- preface can be more detailed and have more you-language. more personable
- edit: what can you edit? everything? be clearer

### PR DESCRIPTION
**Pull request by angnobel**
Fix #222 
Fix #223 
Fix #228

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -18,6 +18,22 @@ HR manager's easy to use features will help you to arrange for upcoming intervie
 The data you have provided will also be stored safely and securely for subsequent uses,
 transferable to other devices too!
 
+### Purpose
+The purpose of this user guide is to give a complete documentation of HR Manger and allow you to easily get started on HR Manager.
+The user guide is also the easiest way for you to troubleshoot any issues and clarify any questions you have with HR Manager.<br>
+Lets simply your HR processes!
```
Comment (sueann-chua) [inline docs/UserGuide.md:24]: Typo: simplify instead of simply

File: docs/UserGuide.md [hunk 2]
```diff
@@ -18,6 +18,22 @@ HR manager's easy to use features will help you to arrange for upcoming intervie
...
+This user guide is a complete guide for all commands and features for HR Manager.
+You can read the whole guide for a comprehensive understanding of the application.
+<br><br>
+Alternatively, you can quickly get started using the [Quick Start](#Quick start) and [Command Summary](#Command summary).<br>
+If you face any issues, you can check the details for each command under the [Features](#features) section. Each section has its own input table that details the requirements and restrictions of each input of that section.
+
+### Complete Product Information
+HR Manger is a simple to use and easy to learn local desktop application that allows you to easily manage your candidates, positions and interviews.
+HR Manager is built on Java and can be run on all major desktop operating systems. HR Manager has a graphic user interface to display information and uses text-based commands to interact with the application.
+
```
Comment (sueann-chua) [inline docs/UserGuide.md:36]: should we combine this part with the intro, I feel like there are some similarities

File: docs/UserGuide.md [hunk 3]
```diff
@@ -18,6 +21,19 @@ HR manager's easy to use features will help you to arrange for upcoming intervie
...
+### Purpose
+The purpose of this user guide is to give a complete documentation of HR Manger and allow you to easily get started on HR Manager.
+The user guide is also the easiest way for you to troubleshoot any issues and clarify any questions you have with HR Manager.<br>
+Lets simplify your HR processes!
+
+### How to use this user guide?
+This user guide is a complete guide for all commands and features for HR Manager.
+You can read the whole guide for a comprehensive understanding of the application.
+<br><br>
+Alternatively, you can quickly get started using the [Quick Start](#Quick start) and [Command Summary](#Command summary).<br>
```
Comment (seaweediman) [inline docs/UserGuide.md:33]: Not sure why but the links do not work.
image

File: docs/UserGuide.md [hunk 4]
```diff
@@ -196,32 +216,41 @@ OR<br>
...
+| **PHONE_NUMBER** | `928389492, 623792` | At least 3 numbers |
+| **ADDRESS** | `Block 123, Woodlands Avenue 1, #09-34` | Any string, any leading or trailing spaces will be removed |
+| **POSITION** | `Software engineer`, `teSTer` | Title of any position that is currently open, case insensitive |
+| **STATUS** | `None`, `Applied`, `Scheduled`, `Interviewed`, `Accepted`, `Rejected`, `Withdrawn` | Must be one of the given examples, case insensitive |
+| **TAG** | `experienced`, `priority` | Must be alphanumeric, without spaces |
+| **INDEX** | `1`,`2` | Must be a positive integer smaller than 2147483647 corresponding to the index of the intended candidate in the <U>currently displayed list of candidate<U> |
+| **REMARK** | `Great Experience`, `@@@ ___ Test Remark` | Any string, leading or trailing spaces are removed|
+> Status for candidates are different then statuses for position amd interviews
```
Comment (seaweediman) [inline docs/UserGuide.md:235]: and instead of amd

**Review by seaweediman — APPROVED** [review]
LGTM

**Pull request merged**